File: src/main/java/com/randomperson22/piggymodtest/npc/georgie/GeorgieNpcModel.java
package com.randomperson22.piggymodtest.npc.georgie;

import net.minecraft.util.ResourceLocation;
import software.bernie.geckolib3.model.AnimatedGeoModel;

public class GeorgieNpcModel extends AnimatedGeoModel<EntityGeorgieNpc> {
	
    public ResourceLocation getModelLocation(EntityGeorgieNpc entity) {
        return new ResourceLocation("piggymodtest", "geo/georgienpc.geo.json");
    }

    public ResourceLocation getTextureLocation(EntityGeorgieNpc entity) {
        return new ResourceLocation("piggymodtest", "textures/entities/georgienpc.png");
    }

    public ResourceLocation getAnimationFileLocation(EntityGeorgieNpc entity) {
        return new ResourceLocation("piggymodtest", "animations/georgienpc.animation.json");
    }
}

File: src/main/kotlin/com/randomperson22/piggymodtest/npc/georgie/GeorgieNpcModel.kt
package com.randomperson22.piggymodtest.npc.georgie

import net.minecraft.util.ResourceLocation
import software.bernie.geckolib3.model.AnimatedGeoModel

class GeorgieNpcModel : AnimatedGeoModel<EntityGeorgieNpc?>() {
    override fun getModelLocation(entity: EntityGeorgieNpc?): ResourceLocation {
        return ResourceLocation("piggymodtest", "geo/georgienpc.geo.json")
    }

    override fun getTextureLocation(entity: EntityGeorgieNpc?): ResourceLocation {
        return ResourceLocation("piggymodtest", "textures/entities/georgienpc.png")
    }

    override fun getAnimationFileLocation(entity: EntityGeorgieNpc?): ResourceLocation {
        return ResourceLocation("piggymodtest", "animations/georgienpc.animation.json")
    }
}